py_gtfs_rt_ingestion/tests/test_configuration.py

```diff
@@ -2,37 +2,41 @@
 
 from datetime import datetime
 from py_gtfs_rt_ingestion import Configuration
+from py_gtfs_rt_ingestion import ConfigType
 
 def test_filname_parsing():
     """
     Check that we are able to get the correct Configuration type for multiple
     filenames
     """
-    trip_updates_type = Configuration.from_filename(
+    trip_updates_type = Configuration(
         '2022-01-01T00:00:02Z_https_cdn.mbta.com_realtime_TripUpdates_enhanced.json.gz')
-    assert trip_updates_type == Configuration.RT_TRIP_UPDATES
+    assert trip_updates_type.config_type == ConfigType.RT_TRIP_UPDATES
 
-    vehicle_positions_type = Configuration.from_filename(
+    vehicle_positions_type = Configuration(
         '2022-01-01T00:00:03Z_https_cdn.mbta.com_realtime_VehiclePositions_enhanced.json.gz')
-    assert vehicle_positions_type == Configuration.RT_VEHICLE_POSITIONS
+    assert vehicle_positions_type.config_type == ConfigType.RT_VEHICLE_POSITIONS
 
-    alerts_type = Configuration.from_filename(
+    alerts_type = Configuration(
         '2022-01-01T00:00:38Z_https_cdn.mbta.com_realtime_Alerts_enhanced.json.gz')
-    assert alerts_type == Configuration.RT_ALERTS
+    assert alerts_type.config_type == ConfigType.RT_ALERTS
 
     with pytest.raises(Exception):
-        Configuration.from_filename('this.is.a.bad.filename.json.gz')
+        Configuration('this.is.a.bad.filename.json.gz')
 
 def test_get_schema():
-    config = Configuration.RT_VEHICLE_POSITIONS
-    schema = config.get_schema()
+    config = Configuration(
+        '2022-01-01T00:00:03Z_https_cdn.mbta.com_realtime_VehiclePositions_enhanced.json.gz')
+    schema = config.export_schema
 
-    with pytest.raises(Exception):
-        unimpled_config = Configuration.RT_ALERTS
-        unimpled_schema = unimpled_config.get_schema()
+    # Requires full implemenation of ConfigDetail for ConfigType
```

Comment:
Remove test, for Configuration class re-factor. Should expect entire ConfigDetail to be implemented, so no exception raised.

Reply:
Any `ConfigDetail` Class that doesn't implement all of the classes defined in the `config_base` ABC should automatically raise an exception when it is invoked. I'm just not sure how exactly we would structure a test for that.